packages/react/src/hooks/actions/__tests__/useAuthSignOutAction.test.ts

```diff
@@ -2,10 +2,12 @@ import { useAuthSignOutAction } from '../useAuthSignOutAction';
 
 import { Auth, Hub } from 'aws-amplify';
 import {
-  ACTIONS_CHANNEL,
   ACTION_AUTH_SIGNOUT_FINISHED,
   ACTION_AUTH_SIGNOUT_STARTED,
+  EVENT_ACTION_AUTH_SIGNOUT,
+  UI_CHANNEL,
 } from '../constants';
+import { AMPLIFY_SYMBOL } from '../../../helpers/constants';
```

Comment:
Should `AMPLIFY_SYMBOL` just be in `constants` rather than `helpers/constants`? It would make these imports a bit cleaner throughout.

Reply:
that's a good idea, I'll clean this up in follow up pr